packages/contracts/contracts/registry/ens/ENSSubdomainRegistrar.sol

```diff
@@ -5,15 +5,17 @@ pragma solidity 0.8.10;
 import "@ensdomains/ens-contracts/contracts/registry/ENS.sol";
 import "@ensdomains/ens-contracts/contracts/resolvers/Resolver.sol";
 
-import "../../core/component/Component.sol";
+import {UUPSUpgradeable} from "@openzeppelin/contracts/proxy/utils/UUPSUpgradeable.sol";
+import {IDAO} from "../../core/IDAO.sol";
+import {PluginUUPSUpgradeable} from "../../core/plugin/PluginUUPSUpgradeable.sol";
+import {DaoAuthorizableUpgradeable} from "../../core/component/DaoAuthorizableUpgradeable.sol";
 
 /// @title ENSSubdomainRegistrar
 /// @author Aragon Association - 2022
 /// @notice This contract registers ENS subdomains under a parent domain specified in the initialization process and maintains ownership of the subdomain since only the resolver address is set. This contract must either be the domain node owner or an approved operator of the node owner. The default resolver being used is the one specified in the parent domain.
-contract ENSSubdomainRegistrar is Component {
-    /// @notice The [ERC-165](https://eips.ethereum.org/EIPS/eip-165) interface ID of the contract.
-    bytes4 internal constant REGISTRY_INTERFACE_ID =
-        this.registerSubnode.selector ^ this.setDefaultResolver.selector;
+contract ENSSubdomainRegistrar is UUPSUpgradeable, DaoAuthorizableUpgradeable {
```

Comment:
I believe that this should be`contract ENSSubdomainRegistrar is PluginUUPSUpgradeable`. 
There was thought put behind making it a `Component`/`Plugin` so that DAOs can reuse it to install it as a plugin to manage ENS domain names. 
What is the need behind removing the `Plugin` status in this PR that is about adding ENS to `PluginRepoRegistry`?